extensions/git/src/decorationProvider.ts

```diff
@@ -78,6 +78,13 @@ class GitIgnoreDecorationProvider implements DecorationProvider {
 		}
 	}
 
+	private isIgnoreConfig(document: TextDocument): boolean {
+		return (
+			document.fileName.endsWith('.gitignore') ||
+			document.fileName.replace(/\\/g, '/').endsWith('.git/info/exclude')
```

Comment:
Are you sure `fileName` includes the full path and is not just the file name?

Reply:
Yes, it's a shorthand for [uri.fsPath](https://github.com/microsoft/vscode/blob/c73b3ca474af58e0ceaf827d5d5bd698b338fe52/src/vs/vscode.d.ts#L105).